pkg/virt-controller/services/template.go

```diff
@@ -623,8 +624,8 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		return nil, err
 	}
 
-	// Register resource requests and limits corresponding to attached multus networks.
-	// TODO(ihar) remove when we adopt Multus mutating webhook that handles the job.
+	// Register resource requests and limits corresponding to attached npwg/multus networks.
+	// TODO(ihar) remove when we adopt Npwg/Multus mutating webhook that handles the job.
```

Comment:
it's really just Multus here :) as in name of the project.

Reply:
done